File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    answer = 0
    answer1 = 0
    for i in range(0, len(num_list), 2):
        answer += num_list[i]
    for j in range(1, len(num_list),2):
        answer1 += num_list[j]
        
    if answer > answer1:
            return answer
    else:
            return answer1

#반복문 밖에서 비교를 해야된다
 

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    let answer = 0;
    let answer1 = 0;
    for (let i = 0; i < num_list.length; i += 2) {
        answer += num_list[i];
    }
    for (let j = 1; j < num_list.length; j += 2) {
        answer1 += num_list[j];
    }
    if (answer > answer1) {
        return answer;
    } else {
        return answer1;
    }
}
